File: testwarez/src/androidTest/java/com/pgssoft/testwarez/test/utils/CucumberTestCase.java
package com.pgssoft.testwarez.test.utils;

import cucumber.api.CucumberOptions;

@CucumberOptions(
        features = "features",
        glue = {"com.pgssoft.testwarez.test.steps"},
        plugin = {"pretty"}
)

public class CucumberTestCase {
}

File: testwarez/src/androidTest/java/com/pgssoft/testwarez/test/test/CucumberTestCase.kt
package com.pgssoft.testwarez.test.test

import cucumber.api.CucumberOptions

@CucumberOptions(features = arrayOf("features"),
        glue = arrayOf("com.pgssoft.testwarez.test.cucumber.steps"),
        plugin = arrayOf("pretty",
                "html:/mnt/sdcard/cucumber-reports/cucumber-html-report",
                "json:/mnt/sdcard/cucumber-reports/cucumber.json",
                "junit:/mnt/sdcard/cucumber-reports/cucumber.xml"),
        tags = arrayOf("@agenda, @test1, @test2"))
class CucumberTestCase